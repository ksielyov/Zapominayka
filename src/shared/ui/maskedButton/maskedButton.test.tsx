import React from 'react';
import renderer from 'react-test-renderer';
import {MaskedButton} from '@ui';
import {LikeButtonComponent} from '@icons';

test('maskedButton should render correctly with likeButtonComponent nested', () => {
  const tree = renderer
    .create(
      <MaskedButton>
        <LikeButtonComponent />
      </MaskedButton>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
