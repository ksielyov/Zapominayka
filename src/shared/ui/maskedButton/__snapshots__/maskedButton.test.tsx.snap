// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`maskedButton should render correctly with likeButtonComponent nested 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#151625",
        "borderRadius": 17.655882352941177,
        "height": 86.31764705882352,
        "position": "absolute",
        "width": 86.31764705882352,
      }
    }
  />
  <RNSVGSvgView
    align="xMidYMid"
    bbHeight={90.24117647058823}
    bbWidth={90.24117647058823}
    fill="#fff"
    focusable={false}
    height={90.24117647058823}
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        Object {
          "flex": 0,
          "height": 90,
          "width": 90,
        },
      ]
    }
    vbHeight={52}
    vbWidth={52}
    width={90.24117647058823}
  >
    <RNSVGGroup
      fill={4294967295}
      propList={
        Array [
          "fill",
        ]
      }
    >
      <RNSVGPath
        clipRule={0}
        d="M25.882 49a98.619 98.619 0 01-3.936-.083c-6.004-.25-13.568-1.09-15.515-3.661C4.7 42.97 3.544 35.929 3.149 29.248c-.057-.978-.099-1.949-.123-2.895v-.027l-.001-.005A86.108 86.108 0 013 24.133c.01-6.11.702-14.782 4.366-17.918C10.123 3.853 18.3 3 25.846 3h.4c2.571.01 5.056.12 7.213.3 6.847.578 11.083 1.794 12.595 3.614 1.041 1.25 1.804 3.77 2.316 7.675a88.516 88.516 0 01.625 11.474v.021c-.002.427-.007.852-.015 1.276-.066 3.753-.329 7.35-.756 10.24-.58 3.936-1.418 6.466-2.56 7.739-2.589 2.891-12.192 3.66-19.576 3.661m0 3c8.348 0 19.213-.869 22.142-4.139 1.29-1.438 2.236-4.298 2.893-8.748.487-3.3.786-7.413.857-11.702.007-.43.012-.862.014-1.295V26.1v-.005a100.049 100.049 0 00-.706-12.995c-.58-4.414-1.44-7.262-2.618-8.675C46.96 2.368 42.172.993 34.432.338A104.916 104.916 0 0025.845 0c-8.536-.001-17.79.962-20.91 3.634C.745 7.22-.007 17.214 0 24.131a97.765 97.765 0 00.03 2.294v.005c.026.98.067 1.981.124 2.991.432 7.626 1.746 15.734 3.724 18.346 2.201 2.908 10.752 3.856 17.54 4.139a111.515 111.515 0 004.66.094m-.196-3h.196-.196z"
        fill={4284964094}
        fillRule={0}
        propList={
          Array [
            "fill",
            "fillRule",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
  <RNSVGSvgView
    align="xMidYMid"
    bbHeight={43.15882352941176}
    bbWidth={43.15882352941176}
    fill="none"
    focusable={false}
    height={43.15882352941176}
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        Object {
          "position": "absolute",
          "zIndex": 1,
        },
        Object {
          "flex": 0,
          "height": 43,
          "width": 43,
        },
      ]
    }
    vbHeight={19}
    vbWidth={23}
    width={43.15882352941176}
  >
    <RNSVGGroup
      fill={null}
      propList={
        Array [
          "fill",
        ]
      }
    >
      <RNSVGPath
        d="M11.234 18.81c-.235 0-.392 0-5.578-2.849C.47 13.111.381 12.973.253 12.77c-.114-.183-.232-2.13-.251-5.751-.02-3.69.123-3.88.286-4.102.114-.151.242-.323 2.91-1.618C5.866.003 6.115 0 6.296 0 6.613 0 6.9.122 9.46 1.523c.638.35 1.47.805 1.865 1.001.414-.197 1.3-.666 1.979-1.022C15.843.164 16.1.055 16.4.055c.22 0 .396 0 3.142 1.428.432.224 2.6 1.35 2.825 1.538.313.26.333.571.355 1.63.014.71.011 1.603.006 2.228-.01 1.198-.04 5.004-.116 5.268-.095.317-.296.447-.79.757-.287.178-.69.423-1.205.73-.943.56-2.224 1.305-3.609 2.094-1.357.782-2.654 1.509-3.642 2.06-1.845 1.023-1.904 1.023-2.133 1.023z"
        fill={4284964094}
        propList={
          Array [
            "fill",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
